sklearn/manifold/tests/test_t_sne.py

```diff
@@ -249,6 +249,7 @@ def test_trustworthiness():
 
 @pytest.mark.parametrize("method", ['exact', 'barnes_hut'])
 @pytest.mark.parametrize("init", ('random', 'pca'))
+@ignore_warnings(category=FutureWarning)  # Delete in 1.2
```

Comment:
I think it's better to be more explicit here. We do not want to catch `FutureWarnings` that are not related to the deprecation.

```suggestion
# TODO: Remove filterwarning in 1.2
@pytest.mark.filterwarnings("ignore:.*TSNE will change from.*:FutureWarning")
@pytest.mark.parametrize("method", ['exact', 'barnes_hut'])
@pytest.mark.parametrize("init", ('random', 'pca'))
```